app/code/community/Bolt/Boltpay/Helper/Api.php

```diff
@@ -195,7 +195,7 @@ public function createOrder($reference, $sessionQuoteId = null, $isAjaxRequest =
             if ($parentQuote->isEmpty() ) {
                 throw new Exception("The parent quote ". $immutableQuote->getParentQuoteId() ." is unexpectedly missing.");
             } else if (!$parentQuote->getIsActive() ) {
-                throw new Exception("The parent quote ". $immutableQuote->getParentQuoteId() ." is currently being processed or has been processed.");
+                throw new Bolt_Boltpay_DuplicatedTransitionException("The parent quote ". $immutableQuote->getParentQuoteId() ." is currently being processed or has been processed.");
```

Comment:
can we somehow check if this quote is processed with what bolt transaction reference?